apps/server/src/modules/user-import/uc/user-import.uc.ts

```diff
@@ -39,12 +40,16 @@ export class UserImportUc {
 		private readonly userRepo: UserRepo
 	) {}
 
-	private featureEnabled() {
+	private featureEnabled(school: School) {
 		const enabled = Configuration.get('FEATURE_USER_MIGRATION_ENABLED') as boolean;
 		const systemId = Configuration.get('FEATURE_USER_MIGRATION_SYSTEM_ID') as string;
-		if (!enabled || !ObjectId.isValid(systemId)) {
+		if (!ObjectId.isValid(systemId)) {
 			throw new InternalServerErrorException('User Migration not configured');
 		}
+		const isLdapPilotSchool = school.features && school.features.includes(SchoolFeatures.LDAP_UNIVENTION_MIGRATION);
+		if (!enabled && !isLdapPilotSchool) {
+			throw new InternalServerErrorException('User Migration not enabled');
```

Comment:
missing test while usage in another UC in this file?

Reply:
test was added